backend/src/services/career-journey.service.ts

```diff
@@ -0,0 +1,47 @@
+import { getCustomRepository } from 'typeorm';
+import CareerJourneyRepository from '../data/repositories/career-journey.service.repository';
+
+import { CareerJourney } from '../data/entities/career-journey';
+import { badRequestError, successRequest } from '~/common/errors';
+
+export const getAllCareerJourneys = async (
+  userId: string,
+): Promise<CareerJourney[]> => {
+  const careerJourneyRepository = getCustomRepository(CareerJourneyRepository);
+  const companyInstance =
+    await careerJourneyRepository.getAllCareerJourneyByUserId(userId);
+  return companyInstance;
+};
+
+export const createCareerJourney = async (
+  body: CareerJourney,
```

Comment:
better to use 'data' because 'body' relates to HTTP protocol, but for the service layer this is redundant knowledge
```suggestion
  data: CareerJourney,
```